### ISSUE TITLE
DashLord : Scripts et cookie

### ISSUE THREAD

**Issue by jrmlnzz** [betagouv/civilsdeladefense#1367]
![Capture d’écran 2022-08-02 à 14 05 17](https://user-images.githubusercontent.com/88372553/182370463-bbb39b16-0183-49cd-b933-d414c08a3316.png)

Est-ce que t'as une idée sur le sujet, je t'avoue ne jamais avoir traité ce genre de sujet par le passé.

**Comment by sebastiencarceles** [comment]
@jrmlnzz je n'ai pas encore bien compris ce que ça signifie, ni ce qu'on cherche à faire ici. Je suppose qu'on veut que les deux premières lignes du tableau passent en vert ?

**Comment by jrmlnzz** [comment]
Yes carrément ! Est-ce que tu saurais me dire ce que représente les deux cookies dont on parle ici ? Avec "ses" et "id" j'imagine qu'on parle de "session" et "identifiant" mais est-ce que tu arrives à avoir plus de détails ?

**Comment by sebastiencarceles** [comment]
Statut :

- j'ai fait appel à la commu Beta.gouv pour voir comment sécuriser ces cookies, car ce n'est pas documenté
- on sollicite les users pour voir si Matomo est utilisé ou non, car visiblement il a été débranche de 3 instances de prod sur 4

**Comment by jrmlnzz** [comment]
Merci, affaire à suivre donc ! 👀

**Comment by sebastiencarceles** [comment]
@jrmlnzz pas d'info de la part de la communauté pour sécuriser les cookies. Je n'ai pas l'impression que ce soit possible. Des news de ton côté pour savoir si le dernier client utilise Matomo ? Car j'aimerais bien le débrancher et résoudre ainsi le problème.

**Comment by jrmlnzz** [comment]
Pas de nouvelles non plus, tu peux le débrancher je pense.
